Mon, 26 Jun 2017 18:09:11 deleteData.py[line:32] ERROR 1205 (HY000): Lock wait timeout exceeded; try restarting transaction
